shtuff.py

```diff
@@ -112,9 +114,33 @@ def shtuff_new(cmd, newline):
 
     spawn_and_stuff(os.environ['SHELL'], cmd)
 
+def shtuff_has(name):
+    pid_file = get_pid_file(name)
+
+    if not os.path.exists(pid_file):
+        print_target_not_found(name)
+        exit(1)
+
+    try:
+        pid = get_pid_from_file(pid_file)
+    except ValueError:
```

Comment:
This is interesting. I'd like to better understand why `ValueError` is happening here. My understanding is the the pidfiles should either:

1. not exist
2. contain a (possibly stale) pid

Reply:
@FatBoyXPC and I had a conversation about this yesterday.  There's something causing `None` to be stored to file when you manually `exit` a shtuff shell (opposed to just closing the tab).  I didn't like that ValueError catch either, but we figured more attention could be put on garbage cleanup instead, and then we could remove the `try`.

Reply:
Steps to reproduce:

```bash
shell-a $ shtuff as foo
```

Let's pretend the pid is 123:

```bash
shell-b $ cat ~/.local/share/shtuff/foo.pid
123%
shell-b $ shtuff into foo exit
```

Watch as `shell-a` exits

```bash
shell-b $ cat ~/.local/share/shtuff/foo.pid
None%
```

Reply:
I've sent out a PR to fix this behavior over in https://github.com/jfly/shtuff/pull/14.

Reply:
@jfly It was still throwing exception when running `into` on an `exit`ed shtuff receiver.  I rebased and fixed that.   Also saw your comment in the test, and added that graceful "not found" message to the `into` command as well 👌